src/app/public/modules/ag-grid/cell-editors/cell-editor-datepicker/cell-editor-datepicker.component.scss

```diff
@@ -1,20 +1,44 @@
 @import '~@skyux/theme/scss/mixins';
 @import '~@skyux/theme/scss/_compat/mixins';
+@import '~@blackbaud/skyux-design-tokens/scss/themes/modern/variables';
 
-.sky-ag-grid-cell-editor-datepicker {
-  position: relative;
-  top: 1px;
-  left: 1px;
+@include sky-theme-modern {
+  .sky-ag-grid-cell-editor-datepicker {
+    sky-datepicker {
+      input.sky-form-control {
+        border-radius: 0;
+        border: 1px solid $sky-theme-modern-color-blue-74;
+        box-shadow: none;
 
-  sky-datepicker {
-    input.sky-form-control {
-      border: none;
-      box-shadow: none;
-      border-right: 1px solid $sky-border-color-neutral-medium;
+        &:focus {
+          border-width: 2px !important;
+        }
+      }
+
+      .sky-input-group-datepicker-btn {
+        border-radius: 0;
+      }
     }
+  }
+}
+
+.sky-theme-default,
```

Comment:
Why not just make these baseline styles (instead of calling out `.sky-theme-default`)? I'm pretty sure builder will add this class in, but is there ever a scenario where these classes might not be present?